sample-compose-fragments/src/main/res/navigation/nav_graph_molecules.xml

```diff
@@ -115,6 +118,10 @@
         android:id="@+id/warningViewFragment"
         android:label="@string/molecules_warning"
         android:name="uk.gov.hmrc.sample_compose_fragments.presentation.fragment.molecules.WarningViewFragment" />
+    <fragment
+        android:id="@+id/selectRowViewFragment"
+        android:name="uk.gov.hmrc.sample_compose_fragments.presentation.fragment.molecules.SelectRowViewFragment"
+        android:label="@string/molecules_select_row" />
 
```

Comment:
```suggestion

    <fragment
        android:id="@+id/selectRowViewFragment"
        android:name="uk.gov.hmrc.sample_compose_fragments.presentation.fragment.molecules.SelectRowViewFragment"
        android:label="@string/molecules_select_row" />
```

Reply:
Jemma, I didn't get this suggestion, seems the same to me.

Reply:
Sorry, I was just being picky over the spacing, no code changes needed!